deep_quoridor/src/arena.py

```diff
@@ -81,12 +81,12 @@ def __init__(
         self.max_walls = max_walls
         self.step_rewards = step_rewards
         self.game = env(
-            board_size=board_size, max_walls=max_walls, step_rewards=step_rewards
+            board_size=board_size,
+            max_walls=max_walls,
+            step_rewards=step_rewards,
         )
```

Comment:
@muralx not sure what's going on with the formatting, I tried switching the line length to different values and around 95 seemed closer, but there was always something different.  What are your settings?
I currently have what's in the repo, the `ruff.toml` file containing just:
```
line-length = 120
```
BTW, notice that the file is in deep_rabbit_hole, not in deep_quoridor, so maybe if you're opening deep_quoridor directly is not being taken into account?

Reply:
hmmm.... i don't know if it works if you don't open the root of the repo in vscode (that's what I've been doing, since in the past we've had dependencies between directories (e.g. annotation_utils)